contracts/evoting/controller/action.go

```diff
@@ -632,6 +565,87 @@ func (a *scenarioTestAction) Execute(ctx node.Context) error {
 	return nil
 }
 
+func setupSimpleElection(ctx node.Context, secret kyber.Scalar, proxyAddr1 string,
+	serdecontext serde.Context, electionFac types.ElectionFactory,
+	service ordering.Service) (string, types.Election, []byte, error) {
+
+	fmt.Fprintln(ctx.Out, "Create election")
+
+	// Define the configuration
+	configuration := fake.BasicConfiguration
+
+	createSimpleElectionRequest := ptypes.CreateElectionRequest{
+		Configuration: configuration,
+		AdminID:       "adminId",
+	}
+
+	signed, err := createSignedRequest(secret, createSimpleElectionRequest)
+	if err != nil {
+		return "", types.Election{}, nil, createSignedErr(err)
+	}
+
+	fmt.Fprintln(ctx.Out, "create election js:", signed)
+
+	url := proxyAddr1 + electionPath
+	fmt.Fprintln(ctx.Out, "POST", url)
+
+	resp, err := http.Post(url, contentType, bytes.NewBuffer(signed))
+	if err != nil {
+		return "", types.Election{}, nil, xerrors.Errorf(failRetrieveDecryption, err)
+	}
+
+	if resp.StatusCode != http.StatusOK {
+		buf, _ := io.ReadAll(resp.Body)
+		return "", types.Election{}, nil, xerrors.Errorf(unexpectedStatus, resp.Status, buf)
+	}
+
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return "", types.Election{}, nil, xerrors.Errorf("failed to read the body of the response: %v", err)
+	}
+
+	fmt.Fprintln(ctx.Out, "response body:", string(body))
+
+	resp.Body.Close()
+
+	var electionResponse ptypes.CreateElectionResponse
+
+	err = json.Unmarshal(body, &electionResponse)
+	if err != nil {
+		return "", types.Election{}, nil, xerrors.Errorf("failed to unmarshal create election response: %v - %s", err, body)
+	}
+
+	electionID := electionResponse.ElectionID
+
+	electionIDBuf, err := hex.DecodeString(electionID)
+	if err != nil {
+		return "", types.Election{}, nil, xerrors.Errorf("failed to decode electionID '%s': %v", electionID, err)
+	}
+
+	election, err := getElection(serdecontext, electionFac, electionID, service)
+	if err != nil {
+		return "", types.Election{}, nil, xerrors.Errorf(getElectionErr, err)
+	}
+
+	// sanity check, the electionID returned and the one stored in the election
+	// type must be the same.
+	if election.ElectionID != electionID {
+		return "", types.Election{}, nil, xerrors.Errorf("electionID mismatch: %s != %s", election.ElectionID, electionID)
+	}
+
+	fmt.Fprintf(ctx.Out, "Title of the election: "+election.Configuration.MainTitle)
+	fmt.Fprintf(ctx.Out, "ID of the election: "+election.ElectionID)
+	fmt.Fprintf(ctx.Out, "Status of the election: "+strconv.Itoa(int(election.Status)))
+
+	return electionID, election, electionIDBuf, nil
+}
+
+func logElectionStatus(title, ID, status string) {
+	dela.Logger.Info().Msg("Title of the election : " + title)
+	dela.Logger.Info().Msg("ID of the election : " + ID)
+	dela.Logger.Info().Msg("Status of the election : " + status)
+}
```

Comment:
Just a detail, but seems less repetitive for the rest of the code this way :)
```suggestion
func logElectionStatus(election Election) {
	dela.Logger.Info().Msg("Title of the election : " + election.Configuration.MainTitle)
	dela.Logger.Info().Msg("ID of the election : " + election.ElectionID)
	dela.Logger.Info().Msg("Status of the election : " + strconv.Itoa(int(election.Status)))
}
```